server/services/ChallongeService.js

```diff
@@ -0,0 +1,89 @@
+const util = require('../util.js');
+const logger = require('../log.js');
+
+class ChallongeService {
+    getTournamentsForUser(user) {
+        return new Promise((resolve, reject) => {
+            if(!user.challonge.key) {
+                reject();
+                return;
+            }
+
+            const personal = util.httpRequest(`https://api.challonge.com/v1/tournaments.json?api_key=${user.challonge.key}`, { json: true });
+            const community = user.challonge.subdomain ? util.httpRequest(`https://api.challonge.com/v1/tournaments.json?api_key=${user.challonge.key}&subdomain=${user.challonge.subdomain}`, { json: true }) : [];
+            Promise.all([personal, community])
+                .then(([personal, community]) => {
+                    let final = [...personal.map(x => x.tournament), ...community.map(x => x.tournament)];
+                    final = final.filter(x => x.game_name === 'Keyforge' && x.state !== 'complete');
+                    final = final.sort((a, b) => {
```

Comment:
There's a sortBy in util.js you could use here

Reply:
i'll figure this out in v1.1